src/main/java/org/zone/utils/Messages.java

```diff
@@ -76,7 +87,7 @@ public static Component getNoMessageSet() {
     }
 
     public static Component getFlagMessageView(Component message) {
-        return getMessageTag().append(message);
+        return getMessageTag().append(message.color(NamedTextColor.GOLD));
```

Comment:
If a component comes into a message, you should not be messing with the format of it. Remove the ``.color(NamedTextColor.GOLD)`` and put it where the component is created

Reply:
K

Reply:
can;t bruh

Reply:
Remove it then

Reply:
ok